src/pages/User/components/PullRequests/ShareButtons.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import pullRequestAmount from './pullRequestAmount';
+
+const ShareButtons = ({ username, pullRequestCount }) => (
+  <div className="pb-8 flex justify-center">
+    <div className="p-2" id="twitter-share">
+      <a
+        target="_blank"
+        className="bg-blue text-white rounded px-2 py-1 pointer text-white no-underline text-sm"
+        href={`https://twitter.com/intent/tweet?text=My progress on hacktoberfest ${pullRequestCount} / ${pullRequestAmount}&url=${window.location.origin}/username/${username}&hashtags=hacktoberfest, hacktoberfestchecker`}
+        data-size="large"
+      >
+        <i className="fab fa-twitter fa-lg" />
+        {' '}
+        Tweet
+      </a>
+    </div>
+    <div
+      className="p-2"
+      id="fb-share"
+      data-href={`${window.location.origin}/username/${username}`}
+      data-layout="button"
+      data-size="large"
+    >
+      <a
+        target="_blank"
+        className="fb-xfbml-parse-ignore bg-blue-dark text-white rounded px-2 py-1 pointer text-white no-underline text-sm"
+        href={`https://www.facebook.com/sharer/sharer.php?u=${window.location.origin}/username/${username}`}
```

Comment:
Shall we use the hostname in place of `window.location.origin`?

Reply:
What is the benefit of that? It's more difficult to use such a variable, and it isn't available to the frontend.

Reply:
Consistency I guess 🤷‍♂️ it's no biggie. You'll have to excuse my lack of react knowledge here so apologies if this is a bad question, but we're using it here https://github.com/jenkoian/hacktoberfest-checker/pull/333/files#diff-af68e82a3010f5e2e3da304e5fe5caffR45 so could we not pass it through as a prop?

Reply:
No need for passing it through as a prop, it's available everywhere through `process.env.REACT_APP_HOSTNAME`. Will change it :smile: